.coderabbit.yaml

```
@@ -0,0 +1,60 @@
+# CodeRabbit Configuration for ppanayotov-com
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)